loops-and-arrays.js

```diff
@@ -14,15 +12,21 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 
 
+const names = ["Henk", "Piet", "Fred", "Joop"];
+console.log(names);
+
+for(let i = 0; i < names.length; i++) {
```

Comment:
Volgens de uitwerkingen had de code ook zo er uit mogen zien:

for (let i = 0; i < names.length; i++) {
  names[i] = names[i] + 'je';
}
console.log(names);

Misschien heeft dit te maken met Nova's comment in de uitwerkingen, regel 47:
// als je alleen de uitkomst logt zou je de array niet veranderen, en dat wil je juist wel!